src/components/Login/LoginOTPCard.tsx

```diff
@@ -126,6 +127,8 @@ export const LoginOTPCard: FunctionComponent<LoginOTPCard> = ({
         showAlert(e.alertProps);
       } else if (e instanceof LoginError) {
         showAlert(e.alertProps);
+      } else if (e instanceof AuthInvalidError) {
+        showAlert({ ...e.alertProps, onOk: () => resetStage() });
```

Comment:
`AuthInvalidError` is an instance of `LoginError`, so 
```suggestion
      } else if (e instanceof AuthInvalidError) {
        showAlert({ ...e.alertProps, onOk: () => resetStage() });
      } else if (e instanceof LoginError) {
        showAlert(e.alertProps);
```

Reply:
changed, thanks!!